### PR TITLE
Update operators.rst

### PR DESCRIPTION
**Pull request by aathan**
I am pretty sure the original text referencing a conversion to integer is wrong, but I am only mostly sure my proposed edits are strictly correct.

### CODE REVIEW

File: docs/types/operators.rst [hunk 1]
```diff
@@ -20,6 +20,9 @@ In case one of the operands is a :ref:`literal number <rational_literals>` it is
 (unsigned types of the same bit-width are considered "smaller" than the signed types).
 If both are literal numbers, the operation is computed with arbitrary precision.
 
+Certain imipicit conversions between number types are disallowed, such as implicit
```
Comment (chriseth) [inline docs/types/operators.rst:23]: Does this provide any value if you don't go into detail?

File: docs/types/operators.rst [hunk 2]
```diff
@@ -41,7 +44,8 @@ and 256 exceeds the range allowed for this type.
 
 Another consequence is that an expression like ``1.5 + 1.5`` is valid but ``1.5 + (true ? 1.5 : 2.5)`` is not.
 This is because the former is a rational expression evaluated in unlimited precision and only its final value matters.
-The latter involves a conversion of a fractional rational number to an integer, which is currently disallowed.
+The latter involves a conversion of a fractional rational number to an unsigned fixed point type (``ufixed8x1``, the
```
Comment (chriseth) [inline docs/types/operators.rst:47]: The main problem here is that the implementation of fixed point types in Solidity is not finished. We decided to take them out of the language again, so maybe it is better to not talk about this in detail and just remove fixed point types from the language altogether.

**Comment by chriseth** [comment]
Thank you for all the good suggestions! I think your PRs have value, but maybe you could slim them down a bit, so we can merge those parts that do not require any discussion first?

**Comment by aathan** [comment]
I'm a noob at making doc suggestion PRs. Happy to conform to whatever process is best for your workflow. Just let me know. Can you clarify for example what you mean by "slim them down"?  Do you mean you'd like me to submit each individual paragraph being changed as an individual PR? 

Or do you just literally mean "make your changes shorter?"  <--- that is exactly counter to adding words that yes, lengthen the docs, but hopefully at the same time reduce ambiguity. It has been my experience that both some amount of repetition and some amount of stating things 2-3 different ways, helps humans absorb new information.

**Comment by chriseth** [comment]
I meant that some of your changes are fine as they are and some others are being discussed. Since discussion on wording is usually lengthy on pull requests, you could extract the parts that do not need any discussion and we can merge them right away.

**Comment by aathan** [comment]
Understood. By the way, is there a forum where we could chat about proposed changes *before* I even write one? For example, I'm currently reading some of the docs on the `address` type members where I could add something to the docs, but I'm not sure it's the right place etc and would like to ask first...

**Comment by chriseth** [comment]
Sure! Just come to our chat room as explained in https://docs.soliditylang.org/en/latest/contributing.html#workflow-for-pull-requests

**Comment by leonardoalt** [comment]
The reviews seem to disagree that the changes here are wanted. Closing for now.

**Pull request closed without merging**